src/app/users/users-update/users-update.component.ts

```diff
@@ -55,8 +55,7 @@ export class UsersUpdateComponent implements OnInit {
       firstName: [ '', Validators.required ],
       middleName: '',
       lastName: [ '', Validators.required ],
-      name: '',
-      login: [ { value: '', disabled: true }, Validators.required ],
+      name: [ { value: '', disabled: true }, Validators.required ],
```

Comment:
this field is actually not required as we are appending form value over this.user which already has the name in it.